src/Facebook/GraphNodes/GraphAlbum.php

```diff
@@ -90,17 +90,7 @@ public function getCreatedTime()
     }
 
     /**
-     * Returns the time the album was updated.
-     *
-     * @return \DateTime|null
-     */
-    public function getUpdatedTime()
-    {
-        return $this->getField('updated_time');
-    }
-
-    /**
-     * Returns the description of the album.
+     * Returns the `description` (The description of the album) as string if present.
```

Comment:
take a look about the class and the docs now, I think it is more clear and readable now

what are you think?